src/ScatterChart.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+// use Atk4\Ui\JsExpression;
```

Comment:
```suggestion
use Atk4\Ui\JsExpression;
```

never comment out uses

PHP CS Fixer is smart enought to detect they are used in commneted code

if commented out, PHP CS Fixer does not have a change to remove them if there are not used